Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetSelect.tsx

```diff
@@ -37,8 +37,10 @@ interface IProps {
     SelectedAssets: Array<SystemCenter.Types.DetailedAsset>,
     ShowModal: boolean,
     Type?: 'single' | 'multiple',
-    SessionStorageID?: string, //TODO: Add this functionality to save filters in session storage (gemstone addition)
-    Title?: string
+    StorageID?: string,
+    Title?: string,
+    ChildrenSectionTitle?: string,
```

Comment:
I thought we removed this in gpa-gemstone`?

Reply:
We did, I simply forgot to remove it as an optional argument here

Reply:
You should see a commit with the argument removed pushed.